browser-sign-in/OktaBrowserSignIn/WelcomeViewController.swift

```diff
@@ -37,8 +37,31 @@ final class WelcomeViewController: UIViewController {
             signInButton.isEnabled = false
         }
     }
+
+    // This functions demonstrates how to securely store the given token returned after successfully
+    // authenticating a user in the device keychain using the default implementation provided.
```

Comment:
I have a few suggestions. It might be clearer to make this a header doc comment. Also, I suggested some edits to make it a bit more readable.
```suggestion
    /// This function demonstrates how to securely store the token in the device keychain after a
    /// successful sign in.
```